java/src/org/openqa/selenium/devtools/CdpClientGenerator.java

```diff
@@ -40,6 +40,7 @@
 import java.nio.file.SimpleFileVisitor;
 import java.nio.file.attribute.BasicFileAttributes;
 import java.util.ArrayList;
+import java.util.LinkedHashMap;
```

Comment:
@joerg1985 - Are we only selectively removing references to guava library? I ask because I see references to Immutable map in this class https://github.com/SeleniumHQ/selenium/blob/8393fed02a81b3dabfe4a20f83ee978449b40c22/java/src/org/openqa/selenium/devtools/CdpClientGenerator.java#L31C34-L31C46

Reply:
This is one step in direction of #12737, removing guava from the java client.

The CdpClientGenerator does still use guava to keep this PR small and is not part of the client.